models/_sector/dex/trades/arbitrum/_schema.yml

```diff
@@ -312,5 +312,24 @@ models:
             version:
               - coves1
               - coves2
+
+  - name: ramses_arbitrum_base_trades
+    meta:
+      blockchain: arbitrum
+      sector: dex
+      project: ramses
+      contributors: echovl, discochuck
+    config:
+      tags: [ 'arbitrum', 'dex', 'trades', 'ramses', 'uniswap', 'v2' ]
+    description: "Ramses Arbitrum base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      - check_dex_base_trades_seed:
+          seed_file: ref('ramses_arbitrum_base_trades_seed')
+          filter:
+            version:
               - 1
```

Comment:
We may insert/append `ramses_arbitrum_base_trades` schema at the end of this file, so it will not break the testing version part of `clipper_arbitrum_base_trades`.

Reply:
it is at the end of the file but for some reason `clipper` testing is failing, not sure why.